core/src/main/scala/org/broadinstitute/dsde/rawls/user/UserService.scala

```diff
@@ -912,7 +912,7 @@ class UserService(protected val userInfo: UserInfo, val dataSource: SlickDataSou
             RequestComplete(StatusCodes.Created)
           }
         case None => Future.successful(RequestComplete(ErrorReport(StatusCodes.Forbidden, s"You must be a billing administrator of ${billingAccountName.value} to create a project with it.")))
```

Comment:
Poking around in this a bit more, I think this line could also say, "Billing Account User."